.gitignore

```diff
@@ -2,4 +2,5 @@
 [Oo]bj/
 .vscode
 .DS_Store
-project.lock.json
\ No newline at end of file
+project.lock.json
+/src/ReadLine/*.user
```

Comment:
Just `*.user` should be sufficient and would cover all places it could occur in the future